ROX-18669: Add listening endpoints info (#64308)
openshift/openshift-docs: operating/audit-listening-endpoints.adoc

* In the {product-title-short} web portal, navigate to *Network* -> *Listening Endpoints*.
* Connect to the `ListeningEndpointsService` object in the API. For more information on the API, navigate to *Help* -> *API reference* in the {product-title-short} web portal.

The page provides a list of processes by deployment, with the following information displayed for each process on the list:

* Deployment name
* Cluster
* Namespace
* Count, or the number of processes listening on the ports in the deployment

You can further filter the information displayed on the page by using the filter field and entering individual deployments, namespaces, and clusters.

//"expand icon" is used elsewhere in docs
Click the expand icon at the top of the list to expand all sections for all deployments listed, or click the expand icon on a single deployment line to view additional information about that deployment. The following information is provided:

* Exec file path: Location of the process
* PID: System ID of the process
* Port: Port on which the process is listening
* Protocol: Protocol in use by the process

> * Pod ID: Name of the pod in which the process is contained

Reviewer: ```suggestion
* Pod ID: Name of the pod where the process is contained
```
Just a suggestion for minimalism to consider.
